app/models/stop.rb

```diff
@@ -16,4 +16,9 @@ class Stop < ApplicationRecord
   def original
     twin_stop || self
   end
+
+  def html_id
```

Comment:
Can you add a comment explaining the intent of this function? Also, should we have tests around this function?